types/schema.d.ts

```diff
@@ -2,6 +2,13 @@ import type { UUID } from 'crypto';
 
 export type Season = 'SPRING' | 'SUMMER' | 'FALL' | 'WINTER';
 
+export interface Profile {
```

Comment:
we also want to include `user_id: UUID` 
let's reference the [notion](https://www.notion.so/calblueprint/Typescript-Schema-942a881ff45f408aa0b977d5c3859a5f) as the source of truth for schema!